dd-sdk-android-androidx/src/main/kotlin/com/datadog/android/androidx/tracking/JetpackExtraAttributesProvider.kt

```diff
@@ -0,0 +1,42 @@
+package com.datadog.android.androidx.tracking
+
+import android.view.View
+import androidx.recyclerview.widget.RecyclerView
+import com.datadog.android.rum.tracking.ViewAttributesProvider
+
+/**
+ * Provides extra attributes for the touch target View.
+ * Those special attributes are specifically related with Jetpack components.
+ * Examples:
```

Comment:
This is a Javadoc, so we need to list exhaustively all attributes this provide. If this is already the case, remove the `Examples:` word as it's misleading.